### DISCUSSION TITLE
Pdf is not define shown.

### DISCUSSION THREAD

**Discussion by chyasirabbasjutt** [cu-esiil-edu/2024-data-short-course#107] [Q&A]
Respected Sir @nquarder  and ma'am @eculler 
![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/166647114/a9a65337-f297-4c48-bc85-5dcd319c6ec0)
![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/166647114/29904951-77d9-49dc-a6c0-a00b749b5767)

**Comment by nquarder** [comment]
Hi @chyasirabbasjutt - can you commit your changes and sync so that I can see your work in GitHub? See the little blue circle with the 1 in the middle way over on the left menu in your codespace? Hit that and then LEAVE A MESSAGE, then select Commit Changes, then select Sync. This will save your work and I can then see it in our cu-esiil-edu GitHub org. Thank you, sir! -NQ

**Comment by chyasirabbasjutt** [comment]
I committed the changes. @nquarder

**Reply by nquarder** [reply]
okay @chyasirabbasjutt 

It looks like part of your issue is when you are trying to create a geodataframe of the area outside the fire boundary.

This is from the solutions manual (https://cu-esiil-edu.github.io/esiil-learning-portal/shortcourse/notebooks/05-vegetation/vegetation.html)

```
out_gdf = (
    gpd.GeoDataFrame(geometry=gdf.envelope)
    .overlay(gdf, how='difference'))
```

You will want to modify this code so that the variable names match what you are doing for your analysis of fires in Punjab. 

I would suggest replacing `out_gdf` in Line 1 with something like `punjab_out_gdf`

Then, in Line 2, you need to replace the `gdf` in `geometry=gdf.envelope` with the name of your boundary variable name. In this case, your boundary is called `punjab`

Try this and see it it helps, sir! -NQ

**Comment by nquarder** [comment]
![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/16786791/fe8c12bf-e67e-4518-956d-17df25de76d3)


Also, @chyasirabbasjutt I suggest changing the title of this plot to say Punjab instead of FATA. Nice job with this notebook, sir! -NQ

**Reply by chyasirabbasjutt** [reply]
Thank you so much
